src/main/java/com/mg105/interface_adapters/InputInterpreter.java

```diff
@@ -30,11 +34,15 @@ public InputInterpreter(@NotNull CharacterMover mover, @NotNull Toggler toggler)
     public void interpret(String key) {
         switch (toggler.getCurrentComponent()) {
             case MAP -> {
+                textChanger.setChangeText();
+
                 switch (key) {
                     case "w" -> mover.generateMapMoveBy(new Point(0, -1));
                     case "a" -> mover.generateMapMoveBy(new Point(-1, 0));
                     case "s" -> mover.generateMapMoveBy(new Point(0, 1));
                     case "d" -> mover.generateMapMoveBy(new Point(1, 0));
+
+                    case "k" -> textChanger.setShowControls(true);
```

Comment:
This is pretty cool, but if you close the side window it does not work (the window never comes back).